hack/build-restic.sh

```diff
@@ -50,6 +50,10 @@ fi
 mkdir ${build_path}/restic
 git clone -b v${RESTIC_VERSION} https://github.com/restic/restic.git ${build_path}/restic
 pushd ${build_path}/restic
-go run build.go --goos "${GOOS}" --goarch "${GOARCH}" --goarm "${GOARM}" -o ${restic_bin}
+if [ -z "${GOARM}" ]; then
```

Comment:
Suggest fixing this in the Tiltfile. 
Adding a blank GOARM parameter in the build Restic line should work.

Reply:
Do you mean go build --goarm=none, I think it is not a valid command in Go.

Reply:
That's right, --goarm="" is good. This means that the executable will be built to work on any ARM architecture version.
Change as review, then.